### ISSUE TITLE
Release a new alpha build.

### ISSUE THREAD

**Issue by NicklasWallgren** [vavr-io/vavr#2690]
The release 1.0.0-alpha-4 is currently blocked by a change made in #2665.

When I run `./gradlew release --info`, I get the following error message:

```
> Task :vavr:vavr:initializeSonatypeStagingRepository FAILED
Caching disabled for task ':vavr:vavr:initializeSonatypeStagingRepository' because:
  Build cache is disabled
Task ':vavr:vavr:initializeSonatypeStagingRepository' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
No stagingProfileId set, querying for packageGroup 'io.vavr'
:vavr:vavr:initializeSonatypeStagingRepository (Thread[Execution worker for ':vavr:vavr' Thread 11,5,main]) completed. Took 0.661 secs.

> Task :vavr:runBuildTasks FAILED
:vavr:runBuildTasks (Thread[Execution worker for ':vavr',5,main]) completed. Took 4.099 secs.

> Task :release FAILED
Release process failed, reverting back any changes made by Release Plugin.
Running [git, checkout, gradle.properties] in [/Users/cafebab3/git/vavr-io/vavr]
Running [git, checkout, gradle.properties] produced output: []
:release (Thread[Execution worker for ':',5,main]) completed. Took 40.157 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':vavr:vavr:initializeSonatypeStagingRepository'.
> Failed to load staging profiles, server at https://oss.sonatype.org/service/local/ responded with status code 401

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 41s
1 actionable task: 1 executed
```

**Comment by danieldietrich** [comment]
Ok, we are one step further, the oss sonatype credentials are now found by the build after we merged #2691.

Now there are problems signing the release...

```
> Task :vavr:vavr:signMavenPublication FAILED
Caching disabled for task ':vavr:vavr:signMavenPublication' because:
  Build cache is disabled
Task ':vavr:vavr:signMavenPublication' is not up-to-date because:
  Task has failed previously.
:vavr:vavr:signMavenPublication (Thread[Execution worker for ':vavr:vavr',5,main]) completed. Took 0.004 secs.

> Task :vavr:runBuildTasks FAILED
:vavr:runBuildTasks (Thread[Execution worker for ':vavr',5,main]) completed. Took 16.91 secs.

> Task :release FAILED
Release process failed, reverting back any changes made by Release Plugin.
Running [git, checkout, gradle.properties] in [/Users/daniel/git/vavr-io/vavr]
Running [git, checkout, gradle.properties] produced output: []
:release (Thread[Execution worker for ':',5,main]) completed. Took 1 mins 0.738 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':vavr:vavr:signMavenPublication'.
> Cannot perform signing task ':signMavenPublication' because it has no configured signatory
```

All ssh signatures are in place, I will take a look why this does not work anymore (after migrating to Gradle 7)...

**Comment by danieldietrich** [comment]
Done! Many thanks to all helping hands!

https://search.maven.org/artifact/io.vavr/vavr/1.0.0-alpha-4/jar
